mysql.go

```diff
@@ -89,9 +89,6 @@ func (driver *Driver) Close() error {
 func (driver *Driver) ensureVersionTableExists() error {
 	_, err := driver.db.Exec("CREATE TABLE IF NOT EXISTS " + tableName + " (version bigint not null primary key);")
 
-	if _, isWarn := err.(mysql.MySQLWarnings); err != nil && !isWarn {
-		return err
-	}
```

Comment:
We must not continue if the error is not `nil`. Could you update your code to return the `err` instead?
Thanks

Reply:
done!